### ISSUE TITLE
Legend: Series icon button to change color is inaccessible

### ISSUE THREAD

**Issue by fastfrwrd** [grafana/grafana#119651]
**Steps to reproduce**: View a timeseries panel with a legend, and then click the series-icon. At the bottom of this image, it's the green line next to "A series."

<img width="752" height="330" alt="Image" src="https://github.com/user-attachments/assets/2f7ce8e7-7fd1-4407-9cf6-294aab28eb37" />

**Actual Result**: The button has no screenreader text, so it 

**Expected Result**: The button should have some screenreader indication of what it does (pops open the ad-hoc color picker)

**Relevant WCAG Criteria:** https://dequeuniversity.com/rules/axe/4.11/aria-command-name

**Environment**:
- Grafana version: 12.4.0
- Data source type & version: n/a
- User OS & Browser: Chrome
- Others: n/a
